presto-main/src/main/java/com/facebook/presto/operator/aggregation/builder/SpillableHashAggregationBuilder.java

```diff
@@ -278,9 +278,10 @@ private void rebuildHashAggregationBuilder()
         if (hashAggregationBuilder != null) {
             hashCollisions += hashAggregationBuilder.getHashCollisions();
             expectedHashCollisions += hashAggregationBuilder.getExpectedHashCollisions();
+            hashAggregationBuilder.close();
         }
 
-        this.hashAggregationBuilder = new InMemoryHashAggregationBuilder(
+        hashAggregationBuilder = new InMemoryHashAggregationBuilder(
```

Comment:
this seems unrelated

Reply:
I understand it makes this method coherent, but the commit message all the changes are related with closing, while here is plain refactor.

Anyway, it is a minor thing. Up to you. I would separate this change.

Generaly, I think there is a convention to avoid `this.` prefix if possible (reasonable). Intellij marks that already.